lib/Doctrine/Migrations/Tools/Console/Command/ExecuteCommand.php

```diff
@@ -38,7 +38,10 @@ protected function configure(): void
             ->addArgument(
                 'versions',
                 InputArgument::REQUIRED | InputArgument::IS_ARRAY,
-                'The versions to execute.',
+                sprintf(
+                    'The versions to execute. Use the FQCN of the migration, or the following reserved aliases: %s',
+                    implode(', ', ['first', 'current', 'prev', 'next', 'latest'])
```

Comment:
does it make sense to get this list from the interface somehow?

Reply:
A `getSupportedAliases` method would be easy to impletement, but using it could be trouble, because of the verbatim return, and the `current-1` formats.
Should I go for it? If so, should it be enforced with a `supports` method, or just a simple listing?